packages/widgets/user-profile-widget/src/lib/widget/api/sdk/createUserSdk.ts

```diff
@@ -21,7 +21,21 @@ export const createUserSdk = ({
     return res.json();
   };
 
+  const getCustomAttributes = async (): Promise<CustomAttr[]> => {
+    if (mock) {
+      return [];
```

Comment:
Don't we want to add mock for it?

Reply:
We can't, as we don't know which would be the user's custom fields.